### PR TITLE
Option to automatically open DevTools

### PR DESCRIPTION
**Pull request by atomgomba**
This PR adds a new option to automatically open DevTools in debug mode. In release mode the option is hidden.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
This is nice. 

* I notice that it only opens upon startup.  It is possible to open/close immediately upon toggle? if yes, then the toggle name could simply be "Open DevTools"
* Could this toggle only be visible when it is a debug-build?

**Review by blckmn — APPROVED** [review]

**Pull request merged**
